bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/DecimalType.java

```diff
@@ -143,6 +141,10 @@ public long longValue() {
         return value.longValue();
     }
 
+    protected State delegateStateAs(Class<? extends State> target) {

```

Comment:
although technically correct, "delegate" somehow tends to mislead the kind read to expect the [delegate pattern](https://en.wikipedia.org/wiki/Delegation_pattern) here. How about "defaultConversion" or something similar?